src/Controllers/JavaScriptMessagesController.php

```diff
@@ -336,6 +336,8 @@ private function getMessages(): array
             /* For query editor */
             'strFormatting' => __('Formatting SQL…'),
             'strNoParam' => __('No parameters found!'),
+            'strQueryCopiedSuccess' => __('Query copied successfully'),
+            'strQueryCopyEmpty' => __('Query is empty!'),
```

Comment:
```suggestion
            'strQueryCopyEmpty' => __('There is no query to copy.'),
```
Or you can avoid that and disable the button if there is no query to copy